packages/api/bin/Importer.js

```diff
@@ -56,10 +57,12 @@ class Importer extends Transform {
    * @returns {undefined} undefined
    */
   _final(callback) {
-    Promise.all(this.promises).then(() => {
-      console.log(`Finished restoring ${this.count} records to DynamoDB`);
-      callback();
-    }).catch(callback);
+    return nodeify(this.waitForRestorationToFinish(), callback);
```

Comment:
I'm increasingly less happy with linters explicitly telling you to use a npm module instead.   Just a general comment, not particularly concerned here.

Reply:
In the case of the lodash warnings, I definitely agree. In cases like this, though, I'd rather err on the side of caution. Messing around with Promises and callbacks is error-prone. I'm happy to punt that complexity off to a package in this case.

Reply:
Agreed, this specific case seems fine.